easybuild/easyconfigs/h/HADDOCK/HADDOCK-2.4-2020-09-foss-2019b.eb

```
@@ -0,0 +1,53 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'HADDOCK'
+version = '2.4-2020-09'
+
+homepage = 'https://wenmr.science.uu.nl/haddock2.4/'
+description = """HADDOCK (High Ambiguity Driven protein-protein DOCKing) is an
+information-driven flexible docking approach for the modeling of biomolecular complexes.
+"""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+# HADDOCK must be downloaded manually after requesting access.
+# To obtain HADDOCK, you must fill the HADDOCK license form.
+# https://www.bonvinlab.org/software/haddock2.4/installation/#downloading-haddock
+#
+# haddock2.4-2020-09.tgz
+sources = ['haddock%(version)s.tgz']
+checksums = ['334c45a3eb45473a9ac91f1b1d5687ef60aa4c6ad7b4064859c13c7941cd77ad']
+
+dependencies = [
+    ('libzeep', '3.0.5'),
+    ('bzip2', '1.0.8'),
+    ('zlib', '1.2.11'),
+    ('XZ', '5.2.4'),
+    ('Boost', '1.71.0'),
+    ('CNS', '1.31-linux-x86_64', '', True)
```

```suggestion
    ('CNS', '1.31', '-linux-x86_64', True)
```
i think we should do it like this